website/content/docs/secrets/gcp.mdx

```diff
@@ -232,6 +253,16 @@ token                 ya29.c.ElodBmNPwHUNY5gcBpnXcE4ywG4w1k...
 token_ttl             3599
 ```
 
+```shell-session
```

Comment:
Similar to my suggestion on the API docs, we might want to provide examples of requesting an access token using all three types (e.g., roleset, static-account, impersonated-account)?

Reply:
Added with the api-docs commit